ros2action/setup.py

```diff
@@ -43,6 +43,7 @@
             # 'echo = ros2action.verb.echo:EchoVerb',
             'info = ros2action.verb.info:InfoVerb',
             'list = ros2action.verb.list:ListVerb',
+            'type = ros2action.verb.type:TypeVerb',
             'send_goal = ros2action.verb.send_goal:SendGoalVerb',
```

Comment:
Alphabetical order, please:

```suggestion
            'send_goal = ros2action.verb.send_goal:SendGoalVerb',
            'type = ros2action.verb.type:TypeVerb',
```

Reply:
addressed by https://github.com/ros2/ros2cli/pull/894/commits/55f271b502d7cd83130201a807aa4c17d4847444